### ISSUE TITLE
Error on env var highlighting

### ISSUE THREAD

**Issue by umbopepato** [c4spar/deno-cliffy#31]
Using cliffy `0.7.0`, when I run any command that shows a help message containing environment variables I get the following error:

```
error: Uncaught TypeError: Cannot read property 'length' of undefined
            if ( details.name.length > 3 ) {
                              ^
    at HelpCommand.parseArgsDefinition (https://deno.land/x/cliffy@v0.7.0/packages/command/lib/base-command.ts:780:31)
    at HelpCommand.highlight (https://deno.land/x/cliffy@v0.7.0/packages/command/commands/help.ts:214:21)
    at https://deno.land/x/cliffy@v0.7.0/packages/command/commands/help.ts:159:26
    at Array.map (<anonymous>)
    at getEnvVars (https://deno.land/x/cliffy@v0.7.0/packages/command/commands/help.ts:157:25)
    at renderHelp (https://deno.land/x/cliffy@v0.7.0/packages/command/commands/help.ts:86:39)
    at HelpCommand.getHelp (https://deno.land/x/cliffy@v0.7.0/packages/command/commands/help.ts:198:9)
    at HelpCommand.show (https://deno.land/x/cliffy@v0.7.0/packages/command/commands/help.ts:32:45)
    at VrCommand.help (https://deno.land/x/cliffy@v0.7.0/packages/command/lib/base-command.ts:1181:31)
    at Object.action (https://deno.land/x/cliffy@v0.7.0/packages/command/lib/default-command.ts:22:26)
```

The env variables are declared as as stated in the documentation:

```
VAR_NAME=<value:string>
```

It looks like `parseArgsDefinition()` expects the whole `<value:string>` type declaration as argument, while `envVar.type` only contains `string`.

**Comment by c4spar** [comment]
Hi @umbopepato,
thx for the issue! I can reproduce it and i will fix it quickly.

**Comment by c4spar** [comment]
Hey @umbopepato,
i have fixed the issue in [v0.7.1](https://github.com/c4spar/deno-cliffy/releases/tag/v0.7.1).
Can you confirm that it works now?

**Comment by umbopepato** [comment]
I works, thanks for such a quick response 🙂
